clickhouse/values.yaml

```diff
@@ -48,7 +48,13 @@ clickhouse:
   ## https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#partitions
   ##
   # rollingUpdatePartition:
-
+  
+  # Security Context
+  securityContext:
+    enabled: false
```

Comment:
nitpick: I would get rid of this `enabled` property just by either setting the `securityContext` to something like you did or by setting it empty like so:

```yaml
securityContext: {}
````

Reply:
thanks @baez90 . I like this idea better, thanks for the feedback. I was initially basing it off what I saw with `metrics` section in `values.yaml`

```
metrics:
    enabled: false
    image:
      registry: docker.io
      repository: f1yegor/clickhouse-exporter
      tag: latest
      pullPolicy: IfNotPresent
      port: 9116
```

Would it be better to have the `enabled` property if there were default values that we wanted to provided with the `securityContext`?

Reply:
I agree. Thanks again!